apps/fabric-website/src/styles/_variables.scss

```diff
@@ -74,5 +73,11 @@ $contentPadding-lg: 32px;
 $contentPadding-xl: 40px;
 $contentPaddingBottom: 100px;
 
+// UHF sizing
```

Comment:
It would be good to expand on this comment a little--something along the lines of "Screen sizes that align with the UHF breakpoints"